dvc/pkg/config.py

```diff
@@ -0,0 +1,18 @@
+def get_pkg_settings(config, name):
```

Comment:
I would try to avoid files like this. It feels that all these functions should go to the Repo or the Pkg classes.

Reply:
See https://github.com/iterative/dvc/pull/2012#discussion_r289625617